waku/node/config.nim

```diff
@@ -59,6 +59,20 @@ proc formatListenAddress(inputMultiAdd: MultiAddress): MultiAddress =
     # If MultiAddress contains "0.0.0.0", replace it for "127.0.0.1"
     return MultiAddress.init(inputStr.replace("0.0.0.0", "127.0.0.1")).get()
 
+proc isWsAddress(ma: MultiAddress): bool =
+  let
+    isWs = ma.contains(multiCodec("ws")).get()
+    isWss = ma.contains(multiCodec("wss")).get()
+
+  return isWs or isWss
+
+proc findWsAddress(extMultiAddrs: seq[MultiAddress]): Option[MultiAddress] =
+  for a in extMultiAddrs:
+    if isWsAddress(a):
+      return some(a)
+  
+  return none(MultiAddress)
```

Comment:
I think this can be simpler. Apparently we only need to check where any MA contains ws or wss.

```suggestion
proc containsWsAddress(extMultiAddrs: seq[MultiAddress]): bool =
  return extMultiAddrs.filterIt( it.isWsAddress() ).len > 0
```
Then only use that proc

Reply:
Thank you! I'll add this change now, it's much more elegant :)

My intention of returning the MA instead of a Bool was that no extra computation has to be done to return the MA, so by returning more information "for free" the function could be reused in more cases in the future.

However, using `filterIt` the way you proposed is much better, simpler, and can be used in cases where you need the MA :)